clang/test/Driver/rocm-detect-windows.hip

```diff
@@ -0,0 +1,9 @@
+// REQUIRES: system-windows
```

Comment:
This doesn't require windows, use an explicit triple

Reply:
Also add a clang-cl test?

Reply:
I don't need .cl, as in OpenCL. I mean cl as in clang-cl, the MSVC compatible driver

Reply:
ah ok.  :-)